app/models/formal_group.rb

```diff
@@ -53,7 +53,7 @@ class FormalGroup < Group
   has_attached_file    :logo,
                        url: "/system/groups/:attachment/:id_partition/:style/:filename",
                        styles: { card: "67x67#", medium: "100x100#" },
-                       default_url: 'default-logo-:style.png'
+                       default_url: AppConfig.theme[:default_group_logo_src]
```

Comment:
Hm, do we care that we're losing our default image sizing here?
(ie, normally you'd be able to get `default-logo-small`, `default-logo-medium`, etc.)

How big is the default group logo src image size?

Reply:
Na, we don't care. It's 826 bytes